Makefile

```diff
@@ -87,6 +88,21 @@ tail:
 	@echo "🥫 Reading logs (Apache2, Nginx) …"
 	tail -f logs/**/*
 
+
+#----------#
+# Services #
+#----------#
+rebuild_language:
+	@echo "🥫 Rebuild language"
+	# Run build_lang.pl
+	${DOCKER_COMPOSE} run --rm backend perl -I/opt/product-opener/lib -I/opt/perl/local/lib/perl5 /opt/product-opener/scripts/build_lang.pl
+
+reset_owner:
+	@echo "🥫 reset owner"
+	${DOCKER_COMPOSE} run --rm --user root backend chown www-data:www-data -R /opt/product-opener/ /mnt/podata
```

Comment:
in production, or reasonably big setups with lots of data, this will take forever to run

Reply:
This is a one shot command that you have to launch manually. 

Really meant to fix dev setups. (maybe I should add a comment ?)